module1/homework/grayscaleImages/main.cpp

```diff
@@ -41,10 +42,12 @@ std::array<std::array<uint8_t, 32>, 32> generateNinja() {
 }
 
 int main() {
-    auto ninja = generateNinja();
+    // //commented out for make-build and tests
+    // auto ninja = generateNinja();
     // printMap(ninja);
-    auto compressed = compressGrayscale(ninja);
-    auto decompressed = decompressGrayscale(compressed);
+    // auto compressed = compressGrayscale(ninja);
```

Comment:
Why this code is commented out?

Reply:
Because ninja is 32x32 and functions are using `constexpr size_t height = 160;` and `constexpr size_t width = 240;` defined in compression.hpp file. The compiler does not allow to pass 32x32 img to functions defined in this way.

Reply:
delete comment :)